### PR TITLE
Remove dependencies on broken rails components

### PR DESCRIPTION
**Pull request by ethanmills**
See https://github.com/rails/rails/issues/41750

### CODE REVIEW

**Comment by ethanmills** [comment]
Currently the following depend on the whole of rails:
canonical-rails 
govuk-components 
health_check

**Comment by ethanmills** [comment]
Lint issues are mostly because the behaviour depends on the version of rails 🙄 Will add ignore comments for now

File: config/environments/staging.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  # config.active_storage.service = :local # TODO enable when rails is upgraded
```
Comment (kicferk1) [inline config/environments/staging.rb:40]: Do we even need to enable that? Are we handling any files? Is active storage needed for anything else?

**Review by kicferk1 — APPROVED** [review]

File: config/environments/staging.rb (hunk 1 above)
Comment (ethanmills) [reply config/environments/staging.rb:40]: I think we'll be taking csv uploads, so will need

**Comment by seb-siem-softwire** [comment]
You do what you got to do to get deployments working again - a temporary solution to get things moving again - as long as we go back and undo things - it should be ok. It's hard to believe how one gem broke the internet. Mime has lost its magic

**Review by seb-siem-softwire — APPROVED** [review]

**Review by amazimbe — APPROVED** [review]

**Pull request merged**